python/sglang/srt/managers/detokenizer_manager.py

```diff
@@ -27,6 +27,8 @@
     BatchEmbeddingOut,
     BatchStrOut,
     BatchTokenIDOut,
+    GetMemPoolSizeReq,
```

Comment:
this is not used here?

Reply:
It is used in `get_memory_pool_size()`

Reply:
seems only `GetMemPoolSizeReqOutput` is used in the event_loop in this file